uberfire-extensions/uberfire-security/uberfire-servlet-security/src/main/java/org/uberfire/ext/security/server/BasicAuthSecurityFilter.java

```diff
@@ -83,7 +90,7 @@ public void doFilter(final ServletRequest _request,
         } finally {
             // invalidate session only when it did not exists before this request
             // and was created as part of this request
-            if (session == null) {
+            if (session == null && invalidate) {
```

Comment:
@cristianonicolai I'm not sure about the old uses cases, but I wonder if there is no use case to invalidate==false and session == null (the old ones).

But I think invalidate default true cover this.

Reply:
@ederign true, old cases still covered by the fact that the default still to invalidate. Only when the filter is specifically parametrized to avoid that ( websocket case ) it will be avoided.